### PR TITLE
Temporarily change the default retry policy

### PR DESCRIPTION
**Pull request by kvigen**
Until we turn on circuit breakers by default lets reduce the number of
retries so that we don't overload our system with retries.

### CODE REVIEW

**Comment by rgarcia** [comment]
@kvigen can you change the name of `DefaultRetryPolicy` to `ExponentialRetryPolicy` now that it's not the default? Also, once the JSDoc stuff is merged can you make the corresponding change to the JS client?

otherwise lgtm!

**Comment by kvigen** [comment]
@rgarcia addressed the comments. Do you mind taking one more pass?

File: Makefile [hunk 1]
```diff
@@ -19,7 +19,7 @@ build: hardcoded/hardcoded.go
 test: build generate $(PKGS) js-tests
 
 js-tests:
-	cd test/js && npm install && npm test
+	cd test/js && rm -rf node_modules && npm install && npm test
```
Comment (rgarcia) [inline Makefile]: this might slow things down when developing locally--is it absolutely necessary?

File: clients/js/genjs.go [hunk 2]
```diff
@@ -120,6 +120,24 @@ const defaultRetryPolicy = {
 };
 
 /**
+ * Use this retry policy to retry a request once.
+ * @alias module:{{.ServiceName}}.RetryPolicies.Single
+ */
+const singleRetryPolicy = {
+  backoffs() {
+    return [1000]
```
Comment (rgarcia) [inline clients/js/genjs.go]: missing semicolon

File: clients/js/genjs.go [hunk 3]
```diff
@@ -120,6 +120,24 @@ const defaultRetryPolicy = {
...
+  },
+  retry(requestOptions, err, res) {
+    if (err || requestOptions.method === "POST" ||
+        requestOptions.method === "PATCH" ||
+        res.statusCode < 500) {
+      return false;
+    }
+    return true;
+  },
+}
```
Comment (rgarcia) [inline clients/js/genjs.go]: missing semicolon

File: Makefile (hunk 1 above)
Comment (rgarcia) [inline Makefile]: ahh i see... I think it's fine to leave as is, and if it slows things down significantly we can try just removing node_modules/swagger-test

**Pull request merged**